### ISSUE TITLE
Accessibility: Focus indicator missing [2]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51839]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

There is no visible focus outline on the linked images in the "you may also like" section.

![T008_KBOutlineMissing2](https://github.com/user-attachments/assets/8fb4a71d-f88f-42bc-9f61-bb4ce646a27c)

### Component or Page: 

Page: Cart (Modern)     

### Name affected element:

Under "You may be interested in…" section, product image links

### Relevant code snippets:

`.editor-styles-wrapper .wc-block-grid__products .wc-block-grid__product .wc-block-components-product-image a, .wc-block-components-product-image a {
    border: 0;
    border-radius: inherit;
    box-shadow: none;
    outline: 0;
    text-decoration: none;
    display: block;
}`

### Recommended fix:

Modify the CSS in your stylesheet to ensure the element has a 2px solid outline with a 2px outline offset on focus.
